### PR TITLE
Add instructions for installation on NHS England machines

### LINKED ISSUE #8: Reference instructions for installing on NHS corporate laptops.
People on NHS corporate laptops have been [given specific instructions](https://nhs.sharepoint.com/sites/X26_URPG/SitePages/get-research-and-design-tools.aspx#:~:text=NHS%20UK%20Prototyping%20Kit) (sharepoint link) for installation / permissions. We should consider either linking to the guidance, or including in the rendered instructions.

### PR DESCRIPTION
**Pull request by frankieroberto**
Adds specific instructions for users of NHS England managed Windows laptops, given that these are a core user group.

Based on the content from the [internal guide](https://nhs.sharepoint.com/sites/X26_URPG/SitePages/nhs-prototype-kit.aspx).

Fixes #8.

Not quite sure how best to link this in with the existing installation instructions for Windows and general guide. Have added a link from the Get started page for now but open to better ideas!

➡️ Preview here: https://nhs-prototyp-add-nhs-en-stqwjl.herokuapp.com/install/nhs-england-windows-laptops

### CODE REVIEW

**Comment by frankieroberto** [comment]
@paulmsmith did these instructions work for you? Anything to add?

File: app/views/install/nhs-england-windows-laptops.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1 class="nhsuk-heading-l">Installation on an NHS England Windows laptop</h1>
+
+      <p>If you have a Windows laptop issued by NHS England corporate IT, then you can largely follow the <a href="/install/windows/terminal">install guide for Windows</a> but there are some additional steps.</p>
+
+      <h2 class="nhsuk-heading-m">Install software dependencies</h2>
+
+      <p>The following can be installed from the Company Portal application:</p>
```
Comment (frankieroberto) [inline app/views/install/nhs-england-windows-laptops.html:30]: Not sure where this application actually lives? Is it a web-based thing we could link to, or some kind of app buried in the start menu?

File: app/views/install/nhs-england-windows-laptops.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+      <ul>
+        <li>Terminal</li>
+        <li>Visual Studio Code</li>
```
Comment (frankieroberto) [inline app/views/install/nhs-england-windows-laptops.html:34]: Not sure if it’s called Visual Studio Code or VS Code within the Company Portal application - we should check.

**Review by vickytnz — COMMENTED** [review]
Something that I found from working through the installation instructions with caroline was that people very much just work down the page so they may miss the windows instructions unless there's some form of course correct in the instructions. 

I don't have enough information to quite know what the answer is, but I think that the simple install page can more for the initial 'you need to do some things first'.

How prominent this is may depend on how many people are going to be this user group and from this how much you want to foreground these details - the 3 options I show play with different options (I am maybe erring towards the callout on a page and then still having the existing page but maybe making it being clearly about the 2 setup steps?). 

I also wonder if the ordering is off in that getting the folder sorted is mission critical and then install software knowing what path to use? 

version with large callout
version with header and expanders
version with short link and callout to page

There is also an opportunity to weave it into other pages

advanced page with callout to main page

**Comment by frankieroberto** [comment]
@vickytnz nice. I think the inset text is better than the warning callout, which is maybe a bit much. It could perhaps even work as a plain paragraph (people sometimes ignore inset text as it can be over-used), but I've kept it as inset text for now.

Tweaked the wording and whitespace slightly in 823e94d9fc5d54b18fd98687dc7d619c01f0a009 - and also switched to using an include instead of a custom macro.

**Pull request merged**